internal/builder/packages.go

```diff
@@ -80,7 +80,7 @@ func buildPackage(sourcePath string) error {
 	}
 
 	fmt.Printf("Copy package content (source: %s)\n", sourcePath)
```

Comment:
Should this use the logger?

Reply:
Based on our discussion in https://github.com/elastic/elastic-package/pull/73/files#r475905214, I would consider this to be a logging statement, not part of the output of the command.

Reply:
Fixed